corehq/util/retry_time.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
```

Comment:
I think we want to have `from __future__ import unicode_literals` since I think usernames can have non-ascii characters